examples/Demo/FluentUI.Demo.Client/Documentation/Components/Checkbox/Examples/CheckboxAppearances.razor

```diff
@@ -0,0 +1,4 @@
+﻿<div style="display: flex;flex-direction: column; gap: 12px;">
```

Comment:
Can you use a component to do the same?
`<FluentStack Orientation="Orientation.Vertical">` 

Also in other examples